homeassistant/components/powerwall/__init__.py

```diff
@@ -122,6 +156,7 @@ async def async_update_data():
             POWERWALL_OBJECT: power_wall,
             POWERWALL_COORDINATOR: coordinator,
             POWERWALL_HTTP_SESSION: http_session,
+            PowerwallError: None,
```

Comment:
```suggestion
            POWERWALL_ERROR: None,
```

Looks like `PowerwallError` is an exception?

Reply:
Ups, seems like the IDE autocompleted that the wrong way...